client/landing/stepper/declarative-flow/internals/steps-repository/import/index.tsx

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+import { StepContainer } from '@automattic/onboarding';
+import { useI18n } from '@wordpress/react-i18n';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import DocumentHead from 'calypso/components/data/document-head';
+import { StepPath } from 'calypso/landing/stepper/declarative-flow/internals/steps-repository';
+import { useCurrentRoute } from 'calypso/landing/stepper/hooks/use-current-route';
+import { useSite } from 'calypso/landing/stepper/hooks/use-site';
+import { useSiteSlugParam } from 'calypso/landing/stepper/hooks/use-site-slug-param';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import CaptureStep from 'calypso/signup/steps/import/capture';
+import ListStep from 'calypso/signup/steps/import/list';
+import {
+	ReadyStep,
+	ReadyNotStep,
+	ReadyPreviewStep,
+	ReadyAlreadyOnWPCOMStep,
+} from 'calypso/signup/steps/import/ready';
+import { GoToStep } from 'calypso/signup/types';
+import { getUrlData } from 'calypso/state/imports/url-analyzer/selectors';
+import isAtomicSiteSelector from 'calypso/state/selectors/is-site-automated-transfer';
+import { getFinalImporterUrl } from './helper';
+import { redirect, removeTrailingSlash } from './util';
+import type { Step } from '../../types';
+import './style.scss';
+
+const ImportStep: Step = function ImportStep( props ) {
+	const { __ } = useI18n();
+	const BASE_ROUTE = 'import';
+	const { navigation } = props;
+
+	/**
+	 ↓ Fields
+	 */
+	const siteSlug = useSiteSlugParam();
+	const site = useSite();
+	const currentRoute = useCurrentRoute();
+	const isAtomicSite = useSelector( ( state ) =>
+		isAtomicSiteSelector( state, site?.ID as number )
+	);
```

Comment:
Trying to avoid Calypso state when possible:

```suggestion
	const isAtomicSite = site?.options?.is_automated_transfer;
```